include/job_metadata.h

```diff
@@ -35,10 +35,10 @@ extern bool CronJobCacheValid;
 
 
 /* functions for retrieving job metadata */
-extern void InitializeJobMetadataCache(void);
-extern void ResetJobMetadataCache(void);
-extern List * LoadCronJobList(void);
-extern CronJob * GetCronJob(int64 jobId);
+PGDLLEXPORT extern void InitializeJobMetadataCache(void);
+PGDLLEXPORT extern void ResetJobMetadataCache(void);
+PGDLLEXPORT extern List * LoadCronJobList(void);
+PGDLLEXPORT extern CronJob * GetCronJob(int64 jobId);
```

Comment:
Normally PGDLLEXPORT is only necessary when functions need to be called from another library/extension. Is that the case here?

Reply:
I am checking to see if it will work without.